common/persistence/serialization/persistence_mapper.go

```diff
@@ -152,6 +153,7 @@ func FromInternalWorkflowExecutionInfo(executionInfo *persistence.InternalWorkfl
 		CompletionEventEncoding:            string(common.EncodingTypeEmpty),
 		VersionHistoriesEncoding:           string(common.EncodingTypeEmpty),
 		InitiatedID:                        common.EmptyEventID,
+		FirstExecutionRunID:                MustParseUUID(executionInfo.FirstExecutionRunID),
```

Comment:
Is there any possibility of this being an invalid UUID? I'm always a little scared of using MustParse for places like this

Reply:
Looking below at the builder - from a naïve perspective this looks maybe it could be an invalid uuid, or at least a nil one

Reply:
MustParseUUID works with empty string